### DISCUSSION TITLE
Freac v1.1.7 (20240103)

### DISCUSSION THREAD

**Discussion by stillusingmp3** [enzo1982/freac#585] [General Discussion]
I was suprised to see the CDDB working again.
![Freac v1 1 7 (20240103)](https://github.com/enzo1982/freac/assets/130705373/83ddabbe-27f9-4963-9a94-1f17a6e6c444)

**Comment by Harpper** [comment]
Here is a link to continuous builds of FREAC https://www.freac.org/downloads-mainmenu-33/308-freac-continuous
